Makefile

```diff
@@ -417,8 +417,8 @@ catalog-push: ## Push a catalog image.
 
 .PHONY: container-build
 container-build: ## Build containers
-	make docker-build bundle-build catalog-build catalog-push
+	make docker-build bundle-build 
```

Comment:
I like the comment you added to clarify the dependency, but I think it would be even clearer if we use the target dependencies for this. Something like

```
catalog-build: bundle-push
    catalog-build

catalog-push: catalog-build

container-push:
    make docker-push bundle-push catalog-push
```

I'm not completely convinced because I don't like to be forced to push something, before building something else 🤷, but it is what it is.

Reply:
I like @clobrano suggestion, just with a small remark that the `catalog-build` target should not trigger `bundle-push`. Because the clearer new target below would try to push the same image again (resulting in two constructive pushes of the bundle image, one from `bundle-push` and one from ` catalog-push` and `catalog-build`).
```
container-push:
    make docker-push bundle-push catalog-push
```

Reply:
Good point, but my whole remark was that `catalog-build` **must** trigger `bundle-push`, otherwise there is no sense, but you are right that then `container-push` will rebuild and push the same bundle again.
This bring us back to my main question, why do we have this a problem only now?

Reply:
Uhm, sorry, maybe I've been too picky. The original behavior is already implemented in NHC and FAR

```
// https://github.com/clobrano/node-healthcheck-operator/blob/f833b31de4784750de08cd8df4d55fa52b739f4f/Makefile#L494-L496
.PHONY: container-push
container-push:  ## Push containers (NOTE: catalog can't be build before bundle was pushed)
	make docker-push bundle-push index-build index-push
```

it is ok for me to continue that way

Reply:
Both of the options seem ok to me, even though I lean towards changing push only when the build succeeds